README.md

```diff
@@ -106,18 +106,6 @@ script:
   # etc.
 ```
 
-It might happen that Clippy is not available for a certain nightly release.
-In this case you can try to conditionally install Clippy from the git repo.
-
-```yaml
-language: rust
-rust:
-  - nightly
-before_script:
-   - rustup component add clippy --toolchain=nightly || cargo install --git https://github.com/rust-lang/rust-clippy/ --force clippy
```

Comment:
Hmm, I think this is still useful for people who want to run clippy on the nightly channel instead of beta or stable.

Reply:
As in, this hack *was* useful before it was broken (#3523) for people who didn't want to wait for new rust releases to get clippy updates.

Reply:
removed that commit again, should be fine now :+1:

Reply:
@phansch Is #3523 still meant to fail? Because it does.